apps/passport-server/src/services/telegramService.ts

```diff
@@ -188,6 +188,30 @@ export class TelegramService {
         this.rollbarService?.reportError(e);
       }
     });
+
+    // The "setup" command helps prospective event organizer to collect necessary information.
+    // Currently, this just prints the ID of the channel.
```

Comment:
I am not sure that we agreed on the flow to set up a DMS bot for a telgram channel / hub being a telegram chat bot interaction. Unless that is something we've agreed on, I don't think this makes sense to add. cc @rrrliu

Reply:
I created this mostly for spinning up my own local development. I image we need some form of this functionality because the alternative of fetching bot updates seemed finicky. Correct me if I'm wrong, my impression is that this bot is very alpha and we can be more fluid on iterating stuff. That said, happy to leave this part out or wrap it in a development only clause.

You are right that we haven't agreed on the onboarding experience. We were talking about an Airtable driven experience on Friday.

Reply:
> I image we need some form of this functionality because the alternative of fetching bot updates seemed finicky

I'm not sure what you mean by this - what bot updates would you be fetching and how would that be finicky?

Fwiw, it is possible to get the ID of both a hub and a channel entirely via the telegram web interface.

@robknight wrote up these instructions for how to get the id of a TG hub / channel:

<img width="838" alt="Screenshot 2023-09-11 at 2 30 56 PM" src="https://github.com/proofcarryingdata/zupass/assets/2636237/9397928a-9dba-42ce-9c0e-a9eb39d315c5">

To get the id of a topic in a hub, you can send a message to a topic, copy paste a link to it, and extract the topic id from it. eg.

<img width="1023" alt="Screenshot 2023-09-11 at 2 33 11 PM" src="https://github.com/proofcarryingdata/zupass/assets/2636237/c8dd2690-7732-4625-a4c0-ea6b7175292d">

The link to that message is: https://t.me/c/1581870465/83/156
And thus the topic id is `83`.

If you want to make the process of finding these ids less manual so that you can develop quicker, I'm all for that. If that is what you want to do, I recommend changing this command from `/setup` to `/channelid` (or something similar), and to remove the language in this command that refers to 'setting things up', and replace it with language that specifically refers to the fact that this command is getting you some sort of id.